aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/codegen/actions/StoreActions.kt

```diff
@@ -19,7 +19,7 @@ import com.amplifyframework.statemachine.Action
 import com.amplifyframework.statemachine.codegen.data.AmplifyCredential
 import com.amplifyframework.statemachine.codegen.data.CredentialType
 
-interface StoreActions {
+internal interface StoreActions {
```

Comment:
Please rename this to `CredentialStoreActions` now that you have added CredentialStoreCognitoActions.